docs/adding_tracks.rst

```diff
@@ -6,8 +6,43 @@ Definition
 
 A track describes one or more benchmarking scenarios. Its structure is described in detail in the :doc:`track reference </track>`.
 
-Example track
--------------
+.. _add_track_create_track:
+
+Creating a track from data in an existing cluster
+-------------------------------------------------
+
+If you already have a cluster with data in it you can use the ``create-track`` subcommand of Rally to create a basic Rally track. To create a Rally track with data from the indices ``products`` and ``companies`` that are hosted by a locally running Elasticsearch cluster, issue the following command::
+
+    esrally create-track --track=acme --target-hosts=127.0.0.1: --indices="products,companies" --output-path=~/tracks
+
+If TLS and basic authentication is enabled, also specify :ref:`--client-options <clr_client_options>` and change ``basic_auth_user`` and ``basic_auth_password`` accordingly::
+
+    esrally create-track --track=acme --target-hosts=127.0.0.1:9200 --client-options="timeout:60,use_ssl:true,verify_certs:false,basic_auth_user:'rally',basic_auth_password:'rally-password'" --indices="products,companies" --output-path=~/tracks
```

Comment:
What is we skip `verify_certs:false` here i.e. rely on the default `true`, and use as an example a URL from an imaginary Elastic Cloud cluster (and use e.g. `basic_auth_user:'elastic'`, `basic_auth_password:'a-secret-and-secure-password'`)

Reply:
I am a bit reluctant skipping certificate verification because I expect the default case to be a self-hosted cluster with self-signed certificates.

Reply:
As discussed elsewhere I've enabled certificate verification explicitly in https://github.com/elastic/rally/pull/970/commits/93aa8898aa3069c65b32376f3e803e93fe91f8de.